app/webpacker/controllers/bulk_actions_controller.js

```diff
@@ -0,0 +1,23 @@
+import ApplicationController from "./application_controller";
+
+export default class extends ApplicationController {
+  connect() {
+    super.connect();
+  }
+
+  // abstract
+  confirm(action) {
+    this.stimulate(action, this.getOrdersIds());
+  }
+
+  // private
+  getOrdersIds() {
+    const order_ids = [];
+    document
+      .querySelectorAll("#listing_orders input[name='order_ids[]']:checked")
+      .forEach((checkbox) => {
+        order_ids.push(checkbox.value);
+      });
```

Comment:
Not related to this PR but would `map` work better than `forEach` here? The we don't have to initialise a variable first the push is done for us.

Reply:
Other alternatives which I find clearer:

```js
  getOrdersIds() {
    checkboxes = document.querySelectorAll("#listing_orders input[name='order_ids[]']:checked");
    return Array.prototype.map.call(checkboxes, (checkbox) => checkbox.value)
  }
```

```js
  getOrdersIds() {
    checkboxes = document.querySelectorAll("#listing_orders input[name='order_ids[]']:checked");
    return Array.from(checkboxes, (checkbox) => checkbox.value)
  }
```

Reply:
I choose 

```js
  getOrdersIds() {
    const checkboxes = document.querySelectorAll(
      "#listing_orders input[name='order_ids[]']:checked"
    );
    return Array.from(checkboxes).map((checkbox) => checkbox.value);
  }
```

53badbb5ab